packages/lutris-git/lutris-git.pacscript

```diff
@@ -0,0 +1,75 @@
+pkgname="lutris-git"
+gives="lutris"
+pkgver="0.5.17"
+pkgdesc="Open Gaming Platform"
+url='https://lutris.net'
+license=("GPL-3.0-or-later")
+provides=("${gives}-git")
+conflicts=("${gives}-deb" "${gives}")
+source=("git+https://github.com/lutris/lutris.git")
+sha256sums=("SKIP")
+depends=("cabextract"
+  "curl"
+  "libglib2.0-0"
+  "libgtk-3-0"
+  "hicolor-icon-theme"
+  "mesa-utils"
+  "p7zip"
+  "psmisc"
+  "python3-cairo"
+  "python3-certifi"
+  "python3-dbus"
+  "python3-distro"
+  "python3-gi-cairo"
+  "python3-gi"
+  "python3-lxml"
+  "python3-pillow"
+  "python3-requests"
+  "python3-yaml"
+  "unzip"
+  "libwebkit2gtk-4.1-0"
+  "xdg-desktop-portal"
+  "xdg-utils"
+  "libxrandr2")
+pacdeps=("python3-moddb")
+makedepends=("git" "meson" "gettext" "desktop-file-utils" "appstream-util")
+checkdepends=("libappstream-glib-dev")
+optdepends=('fluidsynth: For games that have MIDI music'
+  'gamemode: Allows games to request a temporary set of optimisations'
+  'gamescope: Draw the game window isolated from your desktop'
+  'gvfs: GVFS backend'
+  'innoextract: Extract Inno Setup installers'
+  'mesa-vulkan-drivers: Vulkan support'
+  'linux-steam-integration: start Steam with LSI'
+  'mangohud: Display the games FPS + other information'
+  'python3-evdev: Controller support'
+  'python3-pefile: Extract icons from Windows executables'
+  'python3-protobuf: BattleNet integration'
+  'python3-pypresence: Discord Rich Presence integration'
+  'libvkd3d1: DirectX 12 support'
+  'vulkan-icd: Vulkan support'
+  'vulkan-tools: Vulkan support'
+  'wine: easiest way to get all the libraries missing from the Lutris runtime'
+  'winetricks: use system winetricks'
+  'libc6:i386: for 32bit games support'
+  'mangohud:i386: Display the games FPS + other information'
+  'libvkd3d1:i386: DirectX 12 support'
+  'mesa-vulkan-drivers:i386: lib32 Vulkan support')
```

Comment:
```bash
[+] INFO: Optional dependencies
[*] WARNING: mangohud:i386, libvkd3d1:i386, gamescope, python3-pypresence, linux-steam-integration does not exist in apt repositories
```

Reply:
```bash
[*] WARNING: gamescope, python3-pypresence, winetricks, linux-steam-integration does not exist in apt repositories
```
hmm im going to look into these

Reply:
```bash
[*] WARNING: winetricks, python3-pypresence, linux-steam-integration does not exist in apt repositories
```
for sid/trixie

Reply:
> for unstable + testing (looks like our CI does not use contrib)

Contrib is not needed for Debian. No deps error @home in Stable or Testing ....

Reply:
I dot it. For ubuntu : optdeps i386 -> i386 is disabled for ubuntu. need to activate arch=("amd64" "i386") ??

Reply:
> python3-pypresence + linux-steam-integration
> 
> these are the issue ones, because they don't exist in any repository
> 
> the other ones are fine because they exist in some and that's why the optdeps warning allows continue and not error

They are not failling. optdepends doesn"t search for them if they doesn't exists isn't it ? (my new commit doesn't give errors, now lutris run well !)

I will work on theses : 

gamescope, python3-pypresence, linux-steam-integration

I don't want to discard some optdepends. I want the full capacity of the package =)

Reply:
Well we should discard the optdeps that do not exist in apt repos, since they are useless. Maybe they can be pacdeps, but they shouldn't be somewhere where they don't exist

Reply:
It's becoming a feature request: optpacdeps 😜

Reply:
I remember also we can use `optdepends_ubuntu` and `optdepends_debian`

Reply:
> I remember also we can use `optdepends_ubuntu` and `optdepends_debian`

Tomorrow I will add gamescope in optdepends_debian

The 2 others I need optpacdeps but this option need to be seted as No by default whereas optdeps are set as Yes.

The good thing is that if the optdepend doesn't exists, it's ignored by default and not available to users.

I think we can go ahead as this and fix when the option will be available.

No doubts also that most people will not install the git version but the -deb one.